firstissue.json

```
@@ -1,1310 +1,1325 @@
-    "unoplatform/uno",
-    "uutils/coreutils",
-    "valhalla/valhalla",
-    "validatorjs/validator.js",
-    "Varying-Vagrant-Vagrants/VVV",
-    "vectordotdev/vector",
-    "vercel/async-retry",
-    "vercel/commerce",
-    "vercel/examples",
-    "vercel/hazel",
-    "vercel/hyper",
-    "vercel/ncc",
-    "vercel/next-learn",
-    "vercel/next.js",
-    "vercel/nextjs-subscription-payments",
-    "vercel/og-image",
-    "vercel/pkg",
-    "vercel/platforms",
-    "vercel/satori",
-    "vercel/serve",
-    "vercel/styled-jsx",
-    "vercel/swr",
-    "vercel/turbo",
-    "vercel/vercel",
-    "vercel/virtual-event-starter-kit",
-    "vesoft-inc/nebula",
-    "vespa-engine/vespa",
-    "vicky002/AlgoWiki",
-    "VictoriaMetrics/VictoriaMetrics",
-    "vimeo/psalm",
-    "virtual-kubelet/virtual-kubelet",
-    "vitessio/vitess",
-    "vmware-tanzu/carvel-ytt",
-    "vmware-tanzu/community-edition",
-    "vmware-tanzu/octant",
-    "vmware-tanzu/sonobuoy",
-    "vmware-tanzu/velero",
-    "vmware/differential-datalog",
-    "vmware/govmomi",
-    "vmware/open-vm-tools",
-    "vmware/photon",
-    "vmware/pyvmomi",
-    "volcano-sh/volcano",
-    "volta-cli/volta",
-    "VSCodeVim/Vim",
-    "vuejs/vue",
-    "vuejs/vue-router",
-    "vuejs/vuepress",
-    "vuetifyjs/vuetify",
-    "VulcanJS/Vulcan",
-    "wagtail/wagtail",
-    "wasm3/wasm3",
-    "WasmEdge/WasmEdge",
-    "wasmerio/wasmer",
-    "wasp-lang/wasp",
-    "weaveworks/eksctl",
-    "weaveworks/ignite",
-    "weaveworks/weave",
-    "web-platform-tests/wpt",
-    "WebAssembly/spec",
-    "WebAssembly/WASI",
-    "webhintio/hint",
-    "webiny/webiny-js",
-    "WeblateOrg/weblate",
-    "wemake-services/wemake-python-styleguide",
-    "werf/werf",
-    "wesnoth/wesnoth",
-    "wger-project/wger",
-    "whatwg/fetch",
-    "whatwg/html",
-    "whitesmith/rubycritic",
-    "winstonjs/winston",
-    "winsw/winsw",
-    "woocommerce/woocommerce",
-    "woodpecker-ci/woodpecker",
-    "wordpress-mobile/WordPress-Android",
-    "wordpress-mobile/WordPress-iOS",
-    "WordPress/gutenberg",
-    "wp-graphql/wp-graphql",
-    "xenia-project/xenia",
-    "xtdb/xtdb",
-    "xtermjs/xterm.js",
-    "xunit/xunit",
-    "yarnpkg/yarn",
-    "Yelp/paasta",
-    "yewstack/yew",
-    "Yoast/wordpress-seo",
-    "yugabyte/yugabyte-db",
-    "zabbix/zabbix",
-    "zalando/skipper",
-    "zaproxy/zaproxy",
-    "zcash/zcash",
-    "zeebe-io/zeebe",
-    "zeek/zeek",
-    "zeit/hyper",
-    "zeit/next.js",
-    "zephyrproject-rtos/zephyr",
-    "zeromicro/go-zero",
-    "zio/zio",
-    "zitadel/zitadel",
-    "zsh-users/zsh-completions",
-    "zsh-users/zsh-syntax-highlighting",
-    "zulip/zulip"
-  ],
-  "labels": [
-    "good first issue",
-    "help wanted"
-  ]
-}
+[
+  {
+    "name": "GitLab",
+    "provider": "gitlab",
+    "repositories": [
+      "gitlab-org/gitlab|278964"
+    ],
+    "labels": [
+      "quick win",
+      "Seeking community contribution"
```

```suggestion
      "quick win",
      "Seeking community contribution",
      "help wanted",
      "good first issue"
```

Help wanted & good first issue are being used by other projects on GitLab

---

Sounds good, I added those 👍